src/Windowing/OpenToolkit.GraphicsLibraryFramework/OpenToolkit.GraphicsLibraryFramework.csproj

```diff
@@ -1,34 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
-    <PropertyGroup>
-      <TargetFrameworks>net472;netcoreapp3.0</TargetFrameworks>
-      <RootNamespace>OpenToolkit.GraphicsLibraryFramework</RootNamespace>
-      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    </PropertyGroup>
-    
-    <ItemGroup>
-      <ProjectReference Include="..\..\OpenToolkit.Core\OpenToolkit.Core.csproj" />
-    </ItemGroup>
-    
-    <ItemGroup>
-      <PackageReference Condition="'$(TargetFramework)' == 'net472'" Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2">
-        <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
-        <PrivateAssets>all</PrivateAssets>
-      </PackageReference>
-      <PackageReference Condition="'$(TargetFramework)' == 'net472'" Include="System.Memory" Version="4.5" />
-      <PackageReference Include="opentoolkit.redist.glfw" Version="3.3.0-pre20190419183326" />
-    </ItemGroup>
-
-    <ItemGroup>
-        <Content Include="OpenToolkit.GraphicsLibraryFramework.dll.config">
-          <PackageCopyToOutput>true</PackageCopyToOutput>
-          <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-          <!-- Make sure the file gets copied for Framework only. -->
-          <PackagePath>contentFiles/any/net472/</PackagePath>
-        </Content>
-    </ItemGroup>
-
-    <Import Project="..\..\..\props\common.props" />
-    <Import Project="..\..\..\props\nuget-common.props" />
-    <Import Project="..\..\..\props\stylecop.props" />
-</Project>
+  <PropertyGroup>
+    <TargetFrameworks>net472;netcoreapp3.0</TargetFrameworks>
+    <RootNamespace>OpenToolkit.GraphicsLibraryFramework</RootNamespace>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\OpenToolkit.Core\OpenToolkit.Core.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Condition="'$(TargetFramework)' == 'net472'" Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2">
```

Comment:
Are they still `-preview`? If not, please upgrade to the next nonpreview

Reply:
This wasnt introduced in this PR so i dont know why you would bring it up, as its outside the scope of this PR. The only reason it came up in the files changed is because my IDE decided to autoformat all the files, a change i should have resolved for most of the .csproj files seems like i missed the spacings in this one tho.

Reply:
Dont you worry. I just look at things and notice things and eventually ask about them. Since this file was touched, I looked at it and noticed that pre-stuff, which shouldnt be here - IMHO, unless necessary